app/components/listing/feedback/FeedbackSteps.module.scss

```diff
@@ -105,7 +105,6 @@
   width: 340px;
   margin-top: 30px;
   @media screen and (max-width: 767px ) {
-    position: absolute;
     bottom: 40px;
```

Comment:
Why was this change made? It's not clear how this relates to the other changes in this PR. Even if it were an intentional change, it's not obvious to a reviewer what behavior this is intended to change.

Also, if you did intentionally make this change, then you should also remove the `bottom: 40px`, since it has no effect when the `position` property is not set to one of the non-default values (`absolute`, `relative`, `fixed`, or `sticky`).